lib/MusicBrainz/Server/Controller/Role/Create.pm

```diff
@@ -83,7 +83,15 @@ role {
 
                 return unless $args{within_dialog};
 
-                $c->stash( dialog_result => $c->json->encode(JSONSerializer->serialize_internal($c, $entity)) );
+                $c->stash(
+                    current_view => 'Node',
+                    component_path => 'forms/DialogResult',
```

Comment:
Where is that called? For entity creation page only?

Reply:
Correct, but more specifically, for entity creation forms *inside a dialog* after the form is submitted from within the dialog.